src/client.js

```diff
@@ -4,8 +4,16 @@ import 'babel-polyfill';
 import { hydrate } from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import { ensureReady, After } from '@jaredpalmer/after';
+import * as OfflinePluginRuntime from 'offline-plugin/runtime';
 import routes from './app/routes';
 
+OfflinePluginRuntime.install({
```

Comment:
Please add tests for this.

Also, if we are only running the server worker on production, it should be wrapped in a conditional.

Reply:
Done 2nd half